src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -52,27 +50,21 @@ public boolean requiresReference() {
     @Override
     protected void runTool(final JavaSparkContext ctx) {
 
-        final JavaRDD<String> inputAssemblies = ctx.textFile(input).cache();
+        final JavaPairRDD<String, ContigsCollection> breakpointIdsToContigsCollection = loadContigsCollectionKeyedByAssemblyId(ctx, input).cache();
 
-        final long numInputPartitions = inputAssemblies.count();
+        final long numInputPartitions = breakpointIdsToContigsCollection.count();
```

Comment:
This is misnamed:  it's the number of assemblies, nothing to do with partitions.